src/features/user/Settings/EditProfile/AddressManagement/UnsetBillingAddress.tsx

```diff
@@ -31,24 +34,23 @@ const UnsetBillingAddress = ({
 }: Props) => {
   const tAddressManagement = useTranslations('EditProfile.addressManagement');
   const tCommon = useTranslations('Common');
-  const { contextLoaded, user, token, logoutUser } = useUserProps();
+  const { contextLoaded, user, token } = useUserProps();
+  const { putApiAuthenticated } = useApi();
   const { setErrors } = useContext(ErrorHandlingContext);
-  const { tenantConfig } = useTenant();
   const [isLoading, setIsLoading] = useState(false);
 
   const unsetAddress = async () => {
     if (!contextLoaded || !user || !token) return;
     setIsLoading(true);
-    const bodyToSend = {
+    const bodyToSend: UnsetBillingAddressApiPayload = {
       type: ADDRESS_TYPE.OTHER,
     };
```

Comment:
Rename `bodyToSend` to something more relevant, maintaining consistency with the approach elsewhere.